prow/jobs/kyma/kyma-integration-with-compass.yaml

```diff
@@ -0,0 +1,77 @@
+---
+test_infra_ref: &test_infra_ref
+  org: kyma-project
+  repo: test-infra
+  path_alias: github.com/kyma-project/test-infra
+
+franpog859_test_infra_ref: &franpog859_test_infra_ref
```

Comment:
Can we change it directly to `test_infra_ref`?
It doesn't make any sense to merge it like this and then change it, if we're anyways waiting for something else to be merged. Issue should be blocked from merging until the other script is in place. Do you have an issue/pr number for this script merge?

Reply:
I assume the connected issue is #1335 with the PR #1337. Is there a reason the script is separate from this PR?

Reply:
Before merging the script I wanted to test it in the Prow environment. This is the way I found. I created jobs with the `optional: true` setup that it won't affect any other PR. The jobs are pointing to the script in my fork so I can change it and test it without merging it to the test-infra master. After the development part it would be merged and this ref would be changed to `test_infra_ref`
Do you have any suggestion to do it in a more elegant way, @tehcyx ?